loan-payoff/V2/REPWRITERSPECS/BANNO.LOAN.PAYOFF.V2.POW

```diff
@@ -0,0 +1,2572 @@
+**  made by unauthorized personnel.
+**
+**  DO NOT MODIFY THIS FILE UNLESS YOU KNOW WHAT YOU'RE DOING!
+**
+**  --------------------------------------------------------------------
+**  Modification History
+**    Ver. 2.0.0  11/10/2025 R. Robison: Original V2 version of loan payoff PO
+**                           Feature set based on: BANNO.LOAN.PAYOFF.V1.POW Ver. 1.1.1  06/19/24
+**    
+*]
+SYMCONNECT
+STATELESS
+
+TARGET=ACCOUNT
+
+DEFINE
+ #INCLUDE "RD.GETDATA.DEF"
+ #INCLUDE "RB.LISTEXPAND.DEF"
+
+ BNOLASTMODDATE            = '11/10/25'
+ BNOLASTMODTIME            = "17:00 MT"
+ BNOPROGRAMNAME            = "BANNO.LOAN.PAYOFF.V2.POW"
+ BNOCFGFILENAME            = "BANNO.LOAN.PAYOFF.V2.CFG"
+ BNOPROGRAMVERSION         = "2.0.0"
+ BNOPROGRAMNOTE1           = "New v2 version of PowerOn"
+ BNOPROGRAMNOTE2           = ""
+
+ BNODEBUGPRINTINFODAYS     = 60
+ BNODEBUGINCLUDEPARAMS     = 1
+
+ STATEGETPRELOADDATA       = "GETPRELOADDATA"
+ STATEPERFORMCALC          = "PERFORMLOANPAYOFFCALC"
+
+[* Error code setup
+*]
+ BNOERRCODECONFIGREAD      = 500
+  BNOERRMODOPEN            = "[configuration file name] open error - [system generated letter file read error message]"
+  BNOERRMODREAD            = "[configuration file name] read error - [system generated letter file read error message]"
+ BNOERRCODECONFIGVALIDATE  = 501
+  BNOERRMODDUPLICATEPARAM  = "Duplicate Param file entry([parameter name])"
+  BNOERRMODINVALIDVALUE    = "Invalid Param Value([parameter name])"
+ BNOERRCODEDUPLOANTYPE     = 502
+  BNOERRMODDUPLOANTYPE     = "Loan Type [4-digit loan type] ([parameter name]))"
+ BNOERRCODEINELIGLOANTYPE  = 503
+  BNOERRMODINELIGLOANTYPE  = "[4-digit loan type]"
+ BNOERRCODEACCTWARNING     = 504
+  BNOERRMODACCTWARNING     = "[3-digit comma separated account warning list]"
+ BNOERRCODELOANWARNING     = 505
+  BNOERRMODLOANWARNING     = "[3-digit comma separated loan warning list]"
+ BNOERRCODELOANPROJECT     = 506
+  BNOERRMODLOANPROJECT     = "[system generated PowerOn function LOANPROJECTCALC error message]"
+ BNOERRCODEFEESPECFILE     = 507
+  BNOERRMODFEESPECFILE     = "[Banno loan payoff fees PowerOn generated error message]"
+ BNOERRCODEBADPAYOFFDATE   = 508
+ BNOERRCODEINVALID         = 509
+
+[* RGUSER Pass variables - passed from UX code
+*]
+ RGLANGUAGE=NUMBER
+ RGMEMBERNUMBER=CHARACTER(10)
+ RGSLID=CHARACTER
+ RGPAYOFFDATE=DATE
+
+[* Main PowerOn variables and data structures for json
+*]
+ INTERESTDESC=CHARACTER(12) ARRAY(8) 
+ INVALIDWARNLIST=CHARACTER
+ LOANFOUND=NUMBER
+ FOUND=NUMBER
+ LOANELIGIBLE=NUMBER
+ CTLLABEL=CHARACTER
+ CTLINFOCOUNT=NUMBER
+ CTLINFO=CHARACTER ARRAY(10)
+ CTLINFOLABEL=CHARACTER ARRAY(10)
+ CTLINFOMAX=10
+ ADDCTLINFO=CHARACTER
+ ADDCTLINFOLABEL=CHARACTER
+ BOATUSERCHAR=CHARACTER
+ VEHLOANFLD=CHARACTER
+ COLLATERALFIELDDATA=CHARACTER
+ CURRENTFIELD=NUMBER
+
+ USERNUMBER=NUMBER
+ USERCHAR=CHARACTER
+ USERDATE=DATE
+ USERRATE=RATE
+ USERCODE=NUMBER
+ USERAMOUNT=MONEY
```

Comment:
make the definition formatting consistent. I.E:

USERAMOUNT=MONEY
or
PAYOFFCALCAMOUNT          = MONEY